source/includes/_embed2.md.erb

```diff
@@ -519,6 +528,7 @@ Event key | Data | Description
 `ada:campaigns:shown` | `campaignKey` - the key of the campaign which triggered the received message | Triggered when the chat application receives a message that originates from a campaign.
 `ada:campaigns:opened` | `campaignKey` - the key of the last campaign shown before chat was opened | Triggered when chat is opened after a proactive campaign has been shown.
 `ada:campaigns:engaged` | `campaignKey` - the key of the last campaign shown before the conversation was engaged | Triggered when the chatter engages a conversation (sends a message) after being shown a campaign in the same session.
+`ada:end_conversation` | `chatter_id` - contains the chat id for the chatter <br/> `event_data` - contains neccessary chat information (i.e. chat start time, chat end time, chat messages `session_id` - contains the session_id for the chat ) | Triggered when the chatter engages a conversation (sends a message) after being shown a campaign in the same session.
```

Comment:
Hey @josephasaad!

I personally don't know what the `session_id` is for or if it's relevant for the client.
We can swap that out for the `chat_transcript` which is something clients are typically more interested in accessing

Reply:
Hey @wole-ada 

I just made the change and replaced session_id with chatter_transcript which will allow the client to see the chat transcript between the user and the bot